cmd/common-main.go

```diff
@@ -49,10 +49,17 @@ func checkUpdate(mode string) {
 
 // Load logger targets based on user's configuration
 func loadLoggers() {
+	if endpoint, ok := os.LookupEnv("MINIO_LOGGER_HTTP_ENDPOINT"); ok {
```

Comment:
We already have support for `http` targets in `config.json`. Why are we allowing an environment variable?

Reply:
This for gateway @kannappanr

Reply:
maybe a comment can be added about this being added mainly for Gateway

Reply:
Okay